src/ai/susi/server/api/aaa/GetAllUsers.java

```diff
@@ -0,0 +1,62 @@
+package ai.susi.server.api.aaa;
+
+import ai.susi.DAO;
+import ai.susi.json.JsonObjectWithDefault;
+import ai.susi.server.*;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
```

Comment:
Should we include those comments stating who wrote something?
This can be seen from the GitHub history as well.

A comment stating what your class does would be good.